sdk/vaa/payloads.go

```diff
@@ -31,6 +39,9 @@ var (
 	ActionRegisterChain      GovernanceAction = 1
 	ActionUpgradeTokenBridge GovernanceAction = 2
 	ActionModifyBalance      GovernanceAction = 3
+
+	// Wormhole relayer governance actions
+	WormholeRelayerSetDefaultRelayProvider GovernanceAction = 3
```

Comment:
Is there a reason you are using 3 rather than starting at 1?

Reply:
Yeah; the actions for the Generic Relayer will be registering chain, upgrading contract, and setting the default relay provider

The idea is to reuse the 'ActionRegisterChain' and 'ActionUpgradeTokenBridge' governance actions (just as NFTbridge does), but then also add an additional one; to distinguish it from those two it has to be '3'.